### DISCUSSION TITLE
Unable to share PyCharm Project on GitHub

### DISCUSSION THREAD

**Discussion by GlobalCreativeApkDev** [community/community#130443] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

Dear Sir/Madam,

Recently when I tried to "Share Project On GitHub" in PyCharm (for one of my Python projects), I got the error "Cannot load information for github.com/GlobalCreativeApkDev:Request response: Access to this site has been restricted. If you believe this is an error, please contact https://support.github.com.". I believe that this is an error, given I still have my GitHub Access Token active. Can you please assist me with this? Thank you.

Kind regards,
GlobalCreativeApkDev

**Comment by GlobalCreativeApkDev** [comment]
<img width="1440" alt="Screenshot 2024-07-01 at 7 37 04 PM" src="https://github.com/community/community/assets/114658744/4b3bc1b4-3ece-417e-ba4d-4eeab62f3864">

Well, for clarity, above is the screenshot of the error. GlobalCreativeApkDev is the username of my GitHub account. I was trying to share a PyCharm project to GitHub. Thanks.

**Reply by AmirSarrafzadeh** [reply, marked as answer]
The Solution that works for me to solve the problem is reinstalling Pycharm IDE. You can also try to update the IDE.

**Reply by GlobalCreativeApkDev** [reply]
Thanks! It works now!

**Reply by AmirSarrafzadeh** [reply]
My Pleasure.

**Reply by ebndev** [reply]
@GlobalCreativeApkDev & @AmirSarrafzadeh thank you both for being a part of the GitHub Community! Glad you were able to get this issue resolved 😄

**Comment by niteesh18433** [comment]
try to re arrange the things and push the code pycharm to Github

**Reply by StephenLonsdale** [reply]
I have two instances of PyCharm CE, one on windows the other on MacOS. I have tried to share projects from both and received the same error message. I have updated the Windows PyCharm instance to the lasted revision but still received the same error message. The Pycharm CE version on MacOS can no longer be updated. On this basis, it is not a problem with the platform or with an update issue with PyCharm CE. Is it a problem with my GitHub site installation? Maybe a setup problem? I received the following email,
Hey StephenLonsdale!

A third-party OAuth application (JetBrains IDE Integration) with gist, read:org, repo, and workflow scopes was recently authorized to access your account.
Visit https://github.com/settings/connections/applications/58566862bd2a5ff748fb for more information.

To see this and other security events for your account, visit https://github.com/settings/security-log

If you run into problems, please contact support by visiting https://github.com/contact

Thanks,
The GitHub Team

Unfortunately, I have yet to manage any viable contact with the GitHub team apart from these support pages. Is it possible to make actual contact with the team?

Thanks in advance

**Comment by Bobfrank15** [comment]
Problem still extant in March 2025

**Comment by JoshuaBosshardt** [comment]
To anyone that has found a resolution for this issue, I am currently facing this myself and having to manually upload each existing file rather than being able to publish with version control is becoming a SIGNIFICANT hassle. Please, someone provide some form of assistance with this. I have already uninstalled and reinstalled PyCharm, I've checked with JetBrains, I've checked all authentication, logged out and logged back in, tried authentication token pathways, it is on host to publish directly with no licensing issues, it's my own code and I can't do anything with it directly. Any guidance would be greatly appreciated.
